pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -151,9 +152,6 @@ func (r *ReconcileWindowsMachineConfig) Reconcile(request reconcile.Request) (re
 			return reconcile.Result{}, fmt.Errorf("error instantiating cloud provider: %v", err)
 		}
 	}
-	if r.k8sclientset == nil {
```

Comment:
I'm removing the check from the reconcile function as this will not change during runtime. I've moved the validation to the `newReconciler` function: https://github.com/openshift/windows-machine-config-operator/pull/35/commits/1c5c2819a6e491fecf7ddc3e464c870bc136fb39#diff-c9e016282d2886f8f7fc2355cc2c1bf4R53